### DISCUSSION TITLE
My PRO badge is not visible

### DISCUSSION THREAD

**Discussion by ishaanbhimwal** [community/community#52704] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I got my Github education pro membership 2 weeks ago and it's valid and verified. Unfortunately my PRO badge is not visible in the profile section. I have tried toggling on and off the badge in profile settings but it doesn't make any difference.

If you go to one of my projects and then hover over the profile, it shows the badge.

![](https://x0.at/BIfp.png)

**Comment by hamdivazim-archive** [comment]
Hi @ishaanbhimwal,

Did you remember to update preferences?

![207037876-7f303cac-1592-4dc9-a572-5ee12823fb3b](https://user-images.githubusercontent.com/74496056/231728558-eed3cf16-5f7d-477d-bbbe-6566b5aa2eb0.png)

If you've tried that and the badge is still not appearing, try following these steps:
* Try clearing your browser's cache. Sometimes old information stored in your cache and cookies can cause the badge to not appear.
* Can you ask others to find out whether they are seeing the same issue? If they are not and they can see the badge, it is likely an issue with your browser.
* Try waiting for a bit - especially if you recently got it. Maybe wait for 24 hours to see if the badge appears on your public profile too.
* Did you get an email confirming that you've got it? If not, maybe wait for that.

If you've tried the above steps but the issue is still not resolved, [contact GitHub support](https://support.github.com/).

**Comment by ishaanbhimwal** [comment]
I have tried all those methods and it's still not coming. If you go to one of my repos and hover over my username there, ishaanbhimwal, you will see the pro badge. But in profile it isn't visible even in different browsers.

**Comment by hamdivazim-archive** [comment]
If you've tried all the above, [contact GitHub support](https://support.github.com/).
